{{cookiecutter.repository_name}}/{{cookiecutter.package_name}}/architecture/model.py

```diff
@@ -11,14 +11,14 @@ class Model(nn.Module):
     def __init__(self):
         super().__init__()
         self.logits = ModuleCompose(
-            nn.Conv2d(1, 32, 3, 1),
```

Comment:
Could be nice to use a torchvision classifier since we are looking at cifar10 now and not mnist?

Reply:
Yeah sure :) Makes more sense with the 0-1 normalization then

Reply:
I think I dislike having pretrained weights for this since we will need to normalize with their weights etc. Feels too specific for a template

Reply:
Okay, I thought it made sense because it feels like we use something pretrained more often than not. But I can see the argument for not having it.

Reply:
Actually after thinking about it I'd prefer having an even smaller model than the current one. Takes less time to run tests and the model will be overwritten anyway.